types/mongoose/index.d.ts

```diff
@@ -1545,6 +1545,9 @@ declare module "mongoose" {
       callback?: (err: any, doc: DocType | null, res: any) => void): DocumentQuery<DocType | null, DocType>;
     findOneAndUpdate(query: any | Query<any>, update: any,
       callback?: (err: any, doc: DocType | null, res: any) => void): DocumentQuery<DocType | null, DocType>;
+    findOneAndUpdate(query: any | Query<any>, update: any,
```

Comment:
This should be just `any` as per README.

Reply:
I was just copying what's there, but I'm happy to update this (and all the other signatures) to be `any`; will push that soon.

Reply:
I figured as much - did the same with my first pull request for Mongoose types. If you have the time and willpower to update the others then it'd be much appreciated - personally I'd only make sure the newly added types are correct. :slightly_smiling_face: